rules/windows/registry/registry_delete/registry_delete_removal_amsi_registry_key.yml

```diff
@@ -1,26 +1,26 @@
-title: Removal Of Amsi Provider Reg Key
+title: Removal Of AMSI Provider Registry Keys
 id: 41d1058a-aea7-4952-9293-29eaaf516465
 status: test
-description: Remove the AMSI Provider registry key in HKLM\Software\Microsoft\AMSI to disable AMSI inspection
+description: Detects the deletion of AMSI providers registry key entries in HKLM\Software\Microsoft\AMSI. This technique could be used by an attacker in order to disable AMSI inspection.
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1562.001/T1562.001.md
     - https://seclists.org/fulldisclosure/2020/Mar/45
 author: frack113
 date: 2021/06/07
-modified: 2022/03/26
+modified: 2023/02/08
 tags:
     - attack.defense_evasion
     - attack.t1562.001
 logsource:
     product: windows
     category: registry_delete
-    definition: key must be add to the sysmon configuration to works
+    definition: 'Requirements: the following registry key must be added to a sysmon configuration or your monitoring solution in order for this rule to work'
```

Comment:
```suggestion
```
this is a general remark for all sysmon configurations.

Reply:
Yes, but this is on specific registries that are usually not monitored by the configs out there